internal/render/helpers.go

```diff
@@ -203,6 +203,15 @@ func toAgeHuman(s string) string {
 	return duration.HumanDuration(time.Since(t))
 }
 
+// ToRestartAge converts pod last restart time to human duration.
+func ToRestartAge(t metav1.Time) string {
```

Comment:
Not sure I understand why we are not reusing ToAge here. Same implementation `<unknown>` is on par if we can not determine the last restart.

Reply:
I had questions about this. But using `ToAge` makes sense also. To be honest I thought that `n/a` would be more readable than a  column with `<unknown>` when the pod does not have restarts.

Reply:
If memory serves... ToAge `<unknown>` is on par with kubectl.